config.yaml

```diff
@@ -28,8 +14,8 @@ options:
     default: ""
     description: >-
       Providing a valid clouds.yaml will enable OpenStack integration. Setting the
-      openstack-clouds-yaml would enable spawning runners on OpenStack, and disable usage of
-      local LXD for runners. The format for the clouds.yaml is described in the docs:
+      openstack-clouds-yaml would enable spawning runners on OpenStack. The format for the clouds.yaml
```

Comment:
I think we should make clear that this option is now required. So I would remove "Setting  openstack-clouds-yaml would enable spawning runners on OpenStack" and also ensure we are going into blocked state if the config is not passed.

Reply:
Updated to make more sense, as as you say, it is now required. There is a test that checks the invalid config because of this (I had to do it while removing code)